pdc_client/__init__.py

```diff
@@ -187,10 +189,10 @@ def obtain_token(self):
 
     def get_paged(self, res, **kwargs):
         """
-        This call is equivalent to ``res(**kwargs)``, only it retrieves all pages
-        and returns the results joined into a single iterable. The advantage over
-        retrieving everything at once is that the result can be consumed
-        immediately.
+        This call is equivalent to ``res(**kwargs)``, if there is no self.page
```

Comment:
@hluk I was assuming in PDC-1234, we will delete get_paged method and merge its codes into __init__, or maybe we will keep get_paged, if that is the case, no problem then